android/libraries/rib-base/src/androidTest/java/com/badoo/ribs/android/lifecycle/PushTwoPopOnePauseAfterPopTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.badoo.ribs.android.lifecycle
+
+import com.badoo.ribs.android.lifecycle.helper.ExpectedState
+import com.badoo.ribs.test.util.runOnMainSync
+
+class PushTwoPopOnePauseAfterPopTest : PushTwoPopOnePauseTest() {
```

Comment:
Order was indeed wrong, fixed.
Naming is difficult. PushTwoPopOne is the base case on which I'm iterating, and I tried to keep these tests close to each other alphabetically. If I follow what you suggest, it will be even more confusing when looking at the package.